datahub-web-react/package.json

```diff
@@ -87,20 +87,20 @@
         "yamljs": "^0.3.0"
     },
     "scripts": {
-        "analyze": "source-map-explorer 'dist/assets/*.js'",
-        "start": "yarn run generate && vite",
-        "ec2-dev": "yarn run generate && CI=true;export CI;vite",
-        "build": "yarn run generate && NODE_OPTIONS='--max-old-space-size=4096 --openssl-legacy-provider' CI=false vite build",
-        "test": "vitest",
-        "generate": "graphql-codegen --config codegen.yml",
+        "analyze": "NODE_OPTIONS='--max-old-space-size=5120 --openssl-legacy-provider' source-map-explorer 'dist/assets/*.js'",
+        "start": "NODE_OPTIONS='--max-old-space-size=5120 --openssl-legacy-provider' yarn run generate && NODE_OPTIONS='--max-old-space-size=5120 --openssl-legacy-provider' vite",
+        "ec2-dev": "NODE_OPTIONS='--max-old-space-size=5120 --openssl-legacy-provider' yarn run generate && CI=true NODE_OPTIONS='--max-old-space-size=5120 --openssl-legacy-provider' vite",
+        "build": "NODE_OPTIONS='--max-old-space-size=5120 --openssl-legacy-provider' yarn run generate && CI=false NODE_OPTIONS='--max-old-space-size=5120 --openssl-legacy-provider' vite build",
```

Comment:
this should only be necessary on the `vite build` step

Reply:
Removed it from several of them. The storybook build causes OOM in Saas. Enabling coverage on the test also triggers OOM so more then just `vite build` is likely warranted.